### PR TITLE
fix: Add missing configuration

### PR DESCRIPTION
**Pull request by kesara**

### CODE REVIEW

File: helm/values.yaml [hunk 1]
```diff
@@ -216,3 +216,5 @@ env:
   SECRET_KEY: "SpAA3v4icREtlC2ND3oZ8JABMEP2Cf1U"
   ALLOWED_HOSTS: "localhost,127.0.0.1"
   WAGTAILADMIN_BASE_URL: "http://localhost:8080/admin"
+  CSRF_TRUSTED_ORIGINS: "http://localhost:8080/,"
+  MATOMO_SITE_ID: 11
```
Comment (microamp) [inline helm/values.yaml:220]: `"11"`?

File: helm/values.yaml [hunk 2]
```diff
@@ -216,3 +216,5 @@ env:
   SECRET_KEY: "SpAA3v4icREtlC2ND3oZ8JABMEP2Cf1U"
   ALLOWED_HOSTS: "localhost,127.0.0.1"
   WAGTAILADMIN_BASE_URL: "http://localhost:8080/admin"
+  CSRF_TRUSTED_ORIGINS: "http://localhost:8080/,"
```
Comment (microamp) [inline helm/values.yaml:219]: No trailing comma unless needed

File: helm/local.py [hunk 3]
```diff
@@ -71,3 +73,6 @@
     },
     "dummy": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"},
 }
+
+IAB_FEED_URL = "https://www.ietf.org/blog/iab/feed"
+IAB_IETF_BLOG_URL = "https://www.ietf.org/blog/iab/"
```
Comment (microamp) [inline helm/local.py:78]: My understanding is that they are already in `ietf/settings/base.py`?

**Review by microamp — APPROVED** [review]
LGTM. Just some comments.

File: helm/values.yaml (hunk 1 above)
Comment (kesara) [reply helm/values.yaml]: Fixed.

File: helm/values.yaml (hunk 2 above)
Comment (kesara) [reply helm/values.yaml]: I've added another label to remind that this is comma-separated.

File: helm/local.py (hunk 3 above)
Comment (kesara) [reply helm/local.py:78]: Good catch.
I'll remove that.

**Pull request merged**